mne/decoding/receptive_field.py

```diff
@@ -0,0 +1,353 @@
+import numpy as np
+from .base import _get_final_est, BaseEstimator, _check_estimator
+
+
+class ReceptiveField(BaseEstimator):
```

Comment:
> yeah, sklearn API requires that the parameters are not changed between init and storage. What you could do however, is make times a dynamic attibrute which is constructed from the tmin, tmax & sfreq param, in which case no underscore is needed.

I don't get it. All of `tmin`, `tmax`, `sfreq`, as well as `times` and `delays` can all be constructed during `__init__`, so they should all be treated the same way, no?